mappings/net/minecraft/block/Block.mapping

```diff
@@ -165,6 +165,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_9533 shouldDropItemsOnExplosion (Lnet/minecraft/class_1927;)Z
 		ARG 1 explosion
 	METHOD method_9534 onUse (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Lnet/minecraft/class_1269;
+		COMMENT Called whenever a player right-clicks (by default) a block. Is called once on the server and once on the client.
+		COMMENT @return  {@link net.minecraft.util.ActionResult#SUCCESS} to make the player swing his hand and not use his equipped item. {@link net.minecraft.util.ActionResult#CONSUME} to make the player not swing his hand but still not use his item. {@link net.minecraft.util.ActionResult#FAIL} or {@link net.minecraft.util.ActionResult#PASS} To not swing the hand and use the equipped item. When FAIL or PASS is returned on the client and the player is not holding an item , {@link net.minecraft.block.Block#onUse(net.minecraft.block.BlockState, net.minecraft.world.World, net.minecraft.util.math.BlockPos, net.minecraft.entity.player.PlayerEntity, net.minecraft.util.Hand, net.minecraft.util.hit.BlockHitResult)}  will be called twice on each side. It is best to return the same value on both the client and the server to prevent desyncs/rubber-banding. 
```

Comment:
- `#PASS} To not swing` has wrong capitalization.
- I would move everything starting from `When FAIL OR PASS is returned...` to paragraphs in the javadoc body.
-  Could you put the non-qualified names in links too ? eg. `{@link net.minecraft.util.ActionResult ActionResult}`
- *Did you just assume the player's gender ?!*

Reply:
Why specify the non qualified name as well?

Reply:
Because qualified name is way too long, so may as well remove it from rendered docs

Reply:
I think IDEA at least shortens the name when rendering it.